web-dev/src/components/UserProfile/index.js

```diff
@@ -23,7 +30,7 @@ const UserProfile = () => {
                         <span className="fw-bold fs-4">{username}'s Account</span>
 
                     </div>
-                    <UserProfileComponent profile={profile}/>
```

Comment:
Try to get profile from getAuth getUserData since its available everywhere.

Reply:
I did not edit this one yet so it still has the old one. This one is for other user's profile, not the current user. So I will have to find user first. Once you fix the navigate profile/userID for the comment section in the detail page, I will change this.